contrib/demo/.startUtils

```diff
@@ -0,0 +1,35 @@
+wait_command() {
+  local command="$1"; shift
+  local wait_seconds="${1:-40}"; shift # 40 seconds as default timeout
```

Comment:
`wait_command` is never called with a second argument, we can just make this a constant value.

Reply:
I now use the argument